packages/optimizely_x/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import provide from '@userdive/provider'
+import { Plugin } from './plugin'
+provide('optimizely', Plugin)
```

Comment:
provide name is `optimizely`!  Not include `X`.

Reply:
👍